amanuensis/crm/hubspot.py

```diff
@@ -5,17 +5,23 @@
 
```

Comment:
Could you move all these functions in hubspotClient and import those back from it?

Reply:
The functions in amanuensis.crm.hubspot are not being referenced.  The new hubspotclient already has similar functions to handle what's in crm.hubspot.  create_user may be the only one missing

Reply:
Ah, I see. Let's keep it as is for now since this ticket is already huge and we are not using that functionality yet. I ll create a new ticket for bringing over the other methods that are in there. They probably will need a little redesign/rethinking.